transformer_engine/pytorch/module.py

```diff
@@ -292,8 +294,10 @@ def set_extra_state(self, state: torch.Tensor) -> None:
             # Initialize before loading
             self.init_fp8_meta_tensors()
             self.fp8_meta["scaling_fwd"].scale.copy_(scale_fwd)
+            self.fp8_meta["scaling_fwd"].scale_inv.copy_(1.0/scale_fwd)
```

Comment:
This is the backwards compatibility path for a previous version and should not be changed. Please remove.

Reply:
Fair enough. Fixed (amended commit 9dac81c).